package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "swagger-parameters",
   "main": "index.js",
-  "version": "1.3.1",
+  "version": "1.3.2",
```

Comment:
Please do not touch the version here, that's my change to make via `npm version`

Reply:
thanks

Reply:
can you also please bump the version on swole? https://github.com/bendrucker/swole/blob/master/package.json#L48

Reply:
By setting `~1.3.0`, swole allows patch versions. So I won't bump swole, it's not necessary or the correct way to handle semver in node. It's up to you as an application to opt into updates, including transitive ones (via your package-lock or shrinkwrap):

https://docs.npmjs.com/cli/update